sbom-utility-scripts/scripts/add-image-reference-script/tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
```

Comment:
Just noticed the GH workflow probably needs an update to run this, similar to https://github.com/konflux-ci/build-tasks-dockerfiles/blob/4770c7c4fedd305291936056392e877260968719/.github/workflows/build-sbom-utility-scripts-image.yml#L48-L52

Reply:
Added